ppr-ui/src/components/tables/SearchedResults.vue

```diff
@@ -70,7 +70,10 @@
             {{ item.vehicleCollateral.make }} {{ item.vehicleCollateral.model }}
           </template>
           <template v-slot:[`item.debtor.personName`]="{ item }">
-            {{ item.debtor.personName.first }} {{ item.debtor.personName.second }} {{ item.debtor.personName.last }}
+            {{ item.debtor.personName.first }}
+            {{ item.debtor.personName.second }}
```

Comment:
is personName.second a real value? Seems odd that we have 'second' and 'middle' ?

Reply:
The Search History table is populating second name. The Search Results table uses middle name. There is an inconsistency in the API with the search criteria using second name. I updated the description for this issue.